crates/wasmtime/src/runtime.rs

```diff
@@ -395,14 +400,18 @@ impl Config {
     ///
     /// By default cache configuration is not enabled or loaded.
     ///
+    /// This method is only available when the `cache` feature of this crate is
+    /// enaled.
+    ///
     /// # Errors
     ///
     /// This method can fail due to any error that happens when loading the file
     /// pointed to by `path` and attempting to load the cache configuration.
     ///
     /// [docs]: https://bytecodealliance.github.io/wasmtime/cli-cache.html
+    #[cfg(feature = "cache")]
```

Comment:
Make sure to also add

```toml
[package.metadata.docs.rs]
all-features = true
```

to the `Cargo.toml` so users actually see these docs.

Reply:
I think I'll probably leave this out for now, but only because the `cache` feature is on-by-default. I think for now we want to avoid pulling in the `lightbeam` feature which is susceptible to nightly breakage.